python/cudf/cudf/benchmarks/get_datasets.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+# Update this to add/remove/change a dataset, using the following format:
+#
+#  comment about the dataset
+#  dataset download URL
+#  destination dir to untar to
+#  blank line separator
+#
+# FIXME: some test data needs to be extracted to "benchmarks", which is
+# confusing now that there's dedicated datasets for benchmarks.
+CUIO_BENCHMARK_DATASET_DATA="
+# 10GB File
+https://rapidsai-data.s3.us-east-2.amazonaws.com/cudf/benchmark/avro_json_datasets.zip
+cudf/benchmarks/cuio_data/
+# ~14s download
+"
```

Comment:
It feels kinda gross to modify bash scripts to control this. Maybe we should make the URL and directory input arguments?

Reply:
Directory path also pre-defined in pytest, so even that might require dynamic input/default.

Reply:
Changes have been made to accept directory path and url.

pytest is also now capable of accepting dataset directory path.